pontoon/teams/templates/teams/includes/projects.html

```diff
@@ -16,12 +16,14 @@
 
   {{ ProjectList.header(request=True) }}
 
+  {% set locale_projects = locale.available_projects_list() %}
+
```

Comment:
We already make this request in the main template, so we could also move the logic for showing and hiding enabled project on first load to JS. But, that wouldn't have any noticable perf impact. And would be a much bigger codebase change. So I'm keeping it as is for now.